CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Change Log
 
+## Version 1.14.2
+* Release date: May 20, 2022
+* Release status: GA
+
+### What's new in 1.14.2
+* Adds ability to create new SQL Docker instances when publishing a SQL database project
```

Comment:
I'd suggest "Adds APIs for managing Azure resources", the note above would go in the SQL database project changelog/readme.